File: read/app/model/Link.php
<?php

namespace app\model;

class Link extends BaseModel
{
    public int $id;
    public string $link;
    public string $email;
    public string $avatar;
    public string $name;
    public string $icon;
    public string $message;
    public string $data;
    public string $created_time;

    const STATUS_ACCEPT = 1;
    const STATUS_REJECT = 2;
}

File: write/models/link.go
package models

import "time"

type Link struct {
	ID          int64     `xorm:"pk autoincr 'id'" json:"id"`
	Link        string    `json:"link"`
	Email       string    `json:"email"`
	Status      int       `json:"status"`
	Avatar      string    `json:"avatar"`
	Name        string    `json:"name"`
	Icon        string    `json:"icon"`
	Message     string    `json:"message"`
	Data        string    `json:"data"`
	CreatedTime time.Time `xorm:"created 'created_time'" json:"created_time"`
}
